scripts/core/ui/components/video.tsx

```diff
@@ -20,8 +26,23 @@ export class VideoComponent extends React.PureComponent<IProps> {
             poster = item.renditions.viewImage.href;
         }
 
-        const videoRenditions = Object.values(item.renditions)
-            .filter((rendition) => rendition.mimetype != null && rendition.mimetype.startsWith('video'));
+        let videoRenditions: Array<IVideoRenditionItem> = [];
+
+        for (const rend of Object.values(item.renditions)) {
+            if (rend.mimetype == null) {
+                continue;
+            } else if (rend.mimetype.toLowerCase() === 'application/x-mpegurl') {
+                return <HLSVideoComponent
+                    poster={poster}
+                    streamUrl={rend.href}
+                    key={rend.href}
+                    width={this.props.width}
+                    height={this.props.height}
```

Comment:
I think it should get width/height from rendition, we might use some preview and not the original size which would be set on the item

Reply:
`HLSVideoComponent` is a custom render for `VideoComponent`, in this case, width and height determined by parent (VideoComponent) as for `video`.

Reply:
yep right, my bad